src/models.py

```diff
@@ -55,48 +55,41 @@
 ELMO_WEIGHTS_PATH = os.path.join(ELMO_SRC_DIR, ELMO_WEIGHTS_NAME)
 
 
-def build_model(args, vocab, pretrained_embs, tasks):
-    '''Build model according to args '''
-
-    # Build embeddings.
-    if args.openai_transformer:
-        # Note: incompatible with other embedders, but logic in preprocess.py
-        # should prevent these from being enabled anyway.
-        from .openai_transformer_lm.utils import OpenAIEmbedderModule
-        log.info("Using OpenAI transformer model; skipping other embedders.")
-        cove_layer = None
-        embedder = OpenAIEmbedderModule(args)
-        d_emb = embedder.get_output_dim()
-    else:
-        # Default case, used for ELMo, CoVe, word embeddings, etc.
-        d_emb, embedder, cove_layer = build_embeddings(args, vocab,
-                                                       tasks, pretrained_embs)
-    d_sent = args.d_hid
-
+def build_sent_encoder(args, vocab, d_emb, tasks, embedder, cove_layer):
```

Comment:
FYI this will conflict with #452 (probably easier to merge that into this than vice-versa?)

Reply:
let's wait to merge #452 since it's approved, and then merge master into this branch

Reply:
Verified that master was merged into branch.